src/OpenFeature.Contrib.Providers.FeatureManagement/README.md

```diff
@@ -0,0 +1,104 @@
+# FeatureManagement .NET Provider
+> **Note**  
+This requires a new feature of the FeatureManagement system, Variants. This feature is still in preview and has not been fully released.
+
+The FeatureManagement Provider allows you to use the FeatureManagement system as an OpenFeature Provider.
+
+# .NET SDK Usage
```

Comment:
Could you please make this an h2 and bump all the titles below by one?

Reply:
Adjusted the sections.